assets/home/tasks-section/ready.js

```diff
@@ -69,22 +88,26 @@ const Ready = ( { onDismiss } ) => {
 				<li>
 					<a
 						className="sensei-home-ready__social-link"
-						href="https://TODO"
+						href={ twitterLink }
+						target="_blank"
+						rel="noreferrer"
 					>
-						<InstagramCircleIcon />
+						<TwitterCircleIcon />
 						<span className="screen-reader-text">
-							{ __( 'Instagram', 'sensei-lms' ) }
+							{ __( 'Twitter', 'sensei-lms' ) }
 						</span>
 					</a>
 				</li>
 				<li>
 					<a
 						className="sensei-home-ready__social-link"
-						href="https://TODO"
+						href={ tumblerLink }
```

Comment:
With the change above...

```suggestion
						href={ tumblrLink }
```